core/workloads/containerized_workload/containerized_workload.md

```diff
@@ -0,0 +1,288 @@
+# Containerized Workload
+A containerized workload is used for long-running workloads in containers. Here is the definition of the containerized workload.
+
+```yaml
+apiVersion: core.oam.dev/v1alpha2
+kind: WorkloadDefinition
+metadata:
+  name: containerizedworkload.core.oam.dev
```

Comment:
```suggestion
  name: containerizedworkloads.core.oam.dev
```

Reply:
why using purls? is it K8s convention? We want to keep the two "names" the same.  Does this mean that the CRD name needs purls too?

Reply:
https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#create-a-customresourcedefinition

> name must match the spec fields below, and be in the form: <plural>.<group>

I agree that we want to keep the two names the same, and we're subject to a Kubernetes constraint here. Kubernetes requires that the CRD defining `kind: ContainerizedWorkload` in `group: core.oam.dev` be named `containerizedworkloads.core.oam.dev` (plural). So if we want to refer to a CRD by name (and thus make getting the CRD way easier for our controllers) we must use the plural name.